services/cd-service/pkg/repository/transformer.go

```diff
@@ -3787,44 +3800,50 @@ func (c *envReleaseTrain) prognosis(
 
 			if err != nil {
 				return ReleaseTrainEnvironmentPrognosis{
-					SkipCause:        nil,
-					Error:            err,
-					FirstLockMessage: "",
-					AppsPrognoses:    nil,
+					SkipCause:     nil,
+					Error:         err,
+					Locks:         nil,
+					AppsPrognoses: nil,
 				}
 			}
 
 			if len(teamLocks) > 0 {
-				// we don't really care about multiple locks, since they all have the same effect, so we just pick one:
-				var firstLock Lock
-				sortedKeys := sorting.SortKeys(teamLocks)
-				for keyIndex := range sortedKeys {
-					key := sortedKeys[keyIndex]
-					firstLock = teamLocks[key]
-					break
+				locksList := []*api.Lock{}
+				sortedKeys := sorting.SortKeys(appLocks)
+				for _, lockId := range sortedKeys {
+					newLock := &api.Lock{
+						Message:   envLocks[lockId].Message,
+						CreatedAt: timestamppb.New(envLocks[lockId].CreatedAt),
```

Comment:
this should be team locks 
```suggestion
						CreatedAt: timestamppb.New(team[lockId].CreatedAt),
```

Reply:
Done